### PR TITLE
explicitly set SYSCONFDIR in TurboVNC easyconfig

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
although stricly speaking, `CMake` is right... 

https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s07.html#etcoptConfigurationFilesForOpt

File: easybuild/easyconfigs/t/TurboVNC/TurboVNC-2.2.3-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -53,6 +53,11 @@ configopts += "-DOPENGL_egl_LIBRARY=$EBROOTLIBGLVND/lib/libEGL.%s " % SHLIB_EXT
 configopts += "-DOPENGL_opengl_LIBRARY=$EBROOTLIBGLVND/lib/libOpenGL.%s " % SHLIB_EXT
 configopts += "-DOPENGL_glx_LIBRARY=$EBROOTLIBGLVND/lib/libGLX.%s " % SHLIB_EXT
 
+# if installdir is under /opt, cmake will set SYSCONFDIR to /etc/installdir instead of installdir/etc
```
Comment (akesandgren) [inline easybuild/easyconfigs/t/TurboVNC/TurboVNC-2.2.3-GCCcore-8.2.0.eb]: The comment about what CMake will do is slightly wrong.
If installdir is under /opt, CMake will set SYSCONFDIR to /etc/opt/installdir

Comment (migueldiascosta) [reply easybuild/easyconfigs/t/TurboVNC/TurboVNC-2.2.3-GCCcore-8.2.0.eb]: you're right that the comment is inconsistent, but then it's the first part that is wrong, it should be "if installdir starts with /opt"

Comment (akesandgren) [reply easybuild/easyconfigs/t/TurboVNC/TurboVNC-2.2.3-GCCcore-8.2.0.eb]: Well both parts then. The text should perhaps be
"If installdir starts with /opt, i.e., /opt/xxx, CMake will set SYSCONFDIR to /etc/opt/xxx"
That way it's easily compared to the actual documentation from CMake and the FHS specification.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/62a1af52f43b42f78f9dca1a4ad58b71 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**